packages/backend-modules/discussions/lib/Notifications.js

```diff
@@ -206,6 +206,7 @@ const submitComment = async (comment, discussion, context, testUsers) => {
       subjectParams,
       isTopLevelComment,
       icon,
+      commentTag,
```

Comment:
`commentTag` does not exist

Maybe use `comment.tags` instead, and join with `,`